### PR TITLE
feat(dedicated-servers): ola enhancements

### PR DESCRIPTION
**Pull request by radireddy**
use aggregation and reset API's and add polling support

closes #MANAGER-5264

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-5264
| License          | BSD 3-Clause

## Description

use aggregation and reset API's and add polling support

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#14278.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.service.js [hunk 1]
```diff
@@ -140,42 +140,42 @@ export default class DedicatedServerInterfacesService {
...
+    }).then((task) => this.waitForTask(serverName, task.data.taskId));
+  }
+
+  waitForTask(serverName, taskId) {
+    return this.Poller.poll(
+      `/dedicated/server/${serverName}/task/${taskId}`,
+      {},
       {
-        virtualNetworkInterface: interfaceToUngroup.id,
+        namespace: 'dedicated.server.interfaces.ola.aggrigation',
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.service.js:161]:
```suggestion
        namespace: 'dedicated.server.interfaces.ola.aggregation',
```

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js [hunk 2]
```diff
@@ -78,30 +78,28 @@ export default class {
...
-          case OLA_MODES.DEFAULT:
-            return this.InterfaceService.setDefaultInterfaces(
-              this.serverName,
-              this.selectedInterfaces[0],
-            );
-          default:
-            return this.$q.when();
-        }
-      })
+    this.configureInterface()
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js:102]: Easier to read and to maintain that way 👍

**Comment by marie-j** [comment]
Polling is never killed if customer leaves the page ?

**Comment by radireddy** [comment]
> Polling is never killed if customer leaves the page ?

We show a loader during polling to keep customer in same page. Yes, we do not cancel polling if the customer decides to leave.

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14359.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#14359.0 ✘  
* 
  * Build ✘

**Review by Jisay — DISMISSED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#14863.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js (hunk 2 above)
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js:102]: nitpicking : you should return this.configureInterface

**Review by JeremyDec — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#14991.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#15453.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#15453.1 ✘  
* 
  * Build ✘

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.controller.js [hunk 3]
```diff
@@ -2,13 +2,16 @@ import get from 'lodash/get';
...
-  constructor($translate, Alerter, DedicatedServerInterfacesService) {
+  constructor($translate, Alerter, CucCloudMessage, DedicatedServerInterfacesService) {
     this.$translate = $translate;
     this.Alerter = Alerter;
+    this.CucCloudMessage = CucCloudMessage;
     this.InterfaceService = DedicatedServerInterfacesService;
   }
 
   $onInit() {
+    console.log('init');
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.controller.js:13]: remove this

**Comment by ovh-cds** [comment]
CDS Report translator-push#11300.0 ✘  
* 
  * push ✘

**Comment by antleblanc** [comment]
@radireddy Does the base branch is the right one?
It seems that there are some extra commits not related to this Pull Request.

**Comment by frenautvh** [comment]
see : https://github.com/ovh/manager/pull/3840

**Pull request closed without merging**